packager/react-packager/src/DependencyResolver/fastfs.js

```diff
@@ -297,7 +297,7 @@ class File {
     Object.keys(this.children).forEach(key => {
       const file = this.children[key];
       if (file.isDir) {
-        files.push(...file.getFiles());
+        file.getFiles().forEach((f) => files.push(f));
```

Comment:
Or... let's just give up on the idea that `files` is a `const` and use `files.concat`...